dist/focus-ring.js

```diff
@@ -265,7 +225,6 @@ document.addEventListener('DOMContentLoaded', function() {
     if (index(el).contains('focus-ring'))
       return;
     index(el).add('focus-ring');
-    el.setAttribute('data-focus-ring-added', '');
```

Comment:
Essentially, we need *some* mechanism (as @bkardell alluded to on the bug) for opting things in to the always-match-focus-ring behaviour which elements on the whitelist get. For the purposes of this polyfill, this is what we came up with. For the pseudo-selector, we need to come up with a good alternative still.